### ISSUE TITLE
Hi @drstrangelooker @AdamMinton ,

### ISSUE THREAD

**Issue by drstrangelooker** [looker-open-source/looker_deployer#149]
Hi @drstrangelooker @AdamMinton ,

I'm getting the same issue as @jtorreio. 

However, it is happening when I run `ldeploy content import`. I get a generic looker_sdk error. As suggested, I tried `gem update gazer ` and this did not resolve the issue.

Here is a screenshot of the error:

<img width="859" alt="image" src="https://github.com/looker-open-source/looker_deployer/assets/20731762/ddc8acc8-62ea-4ac5-a44d-c3658b7f8aff">

I've updated looker_deployer and looker_sdk and it hasn't done the trick either.

This is a high priority issue as we are blocked from deploying any Looker content to our instances. Can you please investigate? 

Please let me know if you need anything from me. Thank you!

_Originally posted by @jacobakh in https://github.com/looker-open-source/looker_deployer/issues/148#issuecomment-1992616066_

**Comment by jacobakh** [comment]
Looker suggests in the API query_id update message that setting the Legacy Feature "Disallow Numeric Query IDs" to OFF should put the Looker instance back to its original state. Doing this has no affect on the issue I mentioned. 

In addition, we are making no calls to the looker-sdk package, just the `ldeploy content import`. 

Cheers

**Comment by drstrangelooker** [comment]
Yes this is a different issue. `ldeploy` uses `looker-sdk`. Can you try updating to the latest version of `looker-sdk` with the command `pip install --upgrade looker-sdk` and try again. I would expect to get a more informative error message from the newest version of the sdk.

**Comment by jacobakh** [comment]
Hi @drstrangelooker ,

As mentioned above, I've already upgraded it. I did it again for safe measure, and I'm getting the same error when I run the ldeploy import command.

**Comment by jacobakh** [comment]
Hi @drstrangelooker @AdamMinton 

Hope all is well. Following up on the above issue- Is there a timeline for the fix? Please let me know if you need anything from me. Cheers

**Comment by aidenmillerbhep** [comment]
NOTE:

For anyone who finds this and is experiencing this issue (using Windows).

This error appears to be associated with the --target-folder argument. Excluding this from your command may work.

Additionally, you can fork the repo and change line 308 in deploy_content.py from:
`args.target_base = args.target_folder.split('/')[0]`
to 
`args.target_base = args.target_folder.split('\\')[0]`

This has solved the issue for me. I would add a commit to the project, but I think that this change would impact mac/unix users. Potentially switching it to args.target_base = args.target_folder.split(os.sep)[0] could solve for both?
